### DISCUSSION TITLE
Any Updated on Accursed Strike?

### DISCUSSION THREAD

**Discussion by mccabefliam** [Truinto/DarkCodex#314] [General]
This has been the item I've been most excited for on the docket for DarkCodex and was wondering if it is still in the plans to implement or if you are mostly done with adding new content.

**Comment by Truinto** [comment]
Got little time for this project. I do intend to implement this feature. But I also need to address a lot of bugs.

**Reply by analysisparalysis23** [reply]
Any chance of it still happening?
